src/FlipMove.js

```diff
@@ -83,10 +98,14 @@ class FlipMove extends Component {
   getPositionDelta(domNode, key) {
     const newBox  = domNode.getBoundingClientRect();
     const oldBox  = this.state[key];
+    const relativeBox = {
+      'top':newBox.top - this.parentBox.top,
+      'left':newBox.left - this.parentBox.left
+    };
 
     return [
-      oldBox.left - newBox.left,
-      oldBox.top  - newBox.top
+      Math.abs(oldBox.left) - Math.abs(relativeBox.left),
+      Math.abs(oldBox.top)  - Math.abs(relativeBox.top)
```

Comment:
Quick question: Why is the `Math.abs` necessary? Seems to me like if the old box was -20px from the parent and the new box is 20px, we want it to move by 40px, not 0px?

Thanks so much for the PR =)